src/http.rs

```diff
@@ -89,7 +90,7 @@ impl HttpClient {
             .send()?
             .text()?;
 
-        let edn_resp = Edn::from_str(&resp);
+        let edn_resp = Edn::from_str(&resp.replace("#inst", ""));
```

Comment:
This bug was introduced before, when we started using `Edn::from_str` directly to convert to type. Edn-rs does not know how to deal with `#inst` as of now...
https://github.com/naomijub/edn-rs/issues/41